README.md

```diff
@@ -6,58 +6,55 @@
 
 Fylla, the Norse word for `complete`, is an autocompletion script generator for the [Thor](whatisthor.com) framework.
 
-It currently generates zsh completion scripts, but will soon have support for bash scripts as well. 
+Fylla currently generates zsh completion scripts. Support for bash scripts will be added.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add the following line to your application's Gemfile:
 
 ```ruby
 gem 'fylla'
 ```
 
-And then execute:
+execute using the following command:
```

Comment:
hmm. this directly links to the block above, so leaving the `and` in there is sort of necessary. If you could find another way to put this bit together that would be better. 

`bundle` is only necessary if you added the gem to the `Gemfile` in other words.